### DISCUSSION TITLE
it informs that my bot is online but my bot does not respond the token is already valid

### DISCUSSION THREAD

**Discussion by zikdobagui123** [anasty17/mirror-leech-telegram-bot#1742] [General]
![image](https://github.com/user-attachments/assets/3798cc77-888e-407f-8aca-9382beb668c5)
![image](https://github.com/user-attachments/assets/ed6b9b5b-216f-4bb8-b7e1-40e70548f23d)

**Comment by anasty17** [comment]
Logs show that bot is not online and u r not using docker
